views/forms/UserProfileForm/UserProfileForm.test.jsx

```diff
@@ -51,30 +51,26 @@ describe('User Profile Form', () => {
           display_name: 'My Full Name',
           mail: 'myEmail@example.test',
           title: 'My Title',
-          department: 'My Department',
-          company: 'My Company',
-          icon: expect.stringContaining('data:image/png;base64'),
+          icon: '',
+          iconFileName: '',
+          iconFile: pngFile,
         },
         expect.anything()
       )
     )
   })
 
-  test('displays errors and does not submit the form with invalid data', async () => {
+  test('does not submit the form with invalid data', async () => {
     const user = userEvent.setup()
-    const onSubmit = jest.fn()
+    const onSubmit = jest.fn((e) => e.preventDefault())
```

Comment:
do we need this change?

Reply:
[0730f7e](https://github.com/AMP-SCZ/dpdash/pull/478/commits/0730f7e99359b47d86b078fb3e0ad15f74bb8fcb)